WordPressVIPMinimum/Sniffs/JS/WindowSniff.php

```diff
@@ -59,10 +59,20 @@ public function register() {
 			'port'     => true,
 			'password' => true,
 		],
-		'name' => true,
+		'name'   => true,
 		'status' => true,
 	];
 
+	/**
+	 * A list of tokens that are allowed in the syntax.
+	 *
+	 * @var array
+	 */
+	private $syntaxTokens = [
+		T_OBJECT_OPERATOR,
+
```

Comment:
Redundant empty line here.

Reply:
We should just add some basic lint rules and fixers so we can skip this type of feedback in code review.

Reply:
Agreed. This project already has it's own [coding standards config file](https://github.com/Automattic/VIP-Coding-Standards/blob/master/.phpcs.xml.dist), though it may not be suitable for this particular issue, since splitting array items by a single line may be desirable for whatever reason.